parachains/runtimes/bridge-hubs/bridge-hub-kusama/src/xcm_config.rs

```diff
@@ -41,10 +40,9 @@ use xcm_executor::{traits::WithOriginFilter, XcmExecutor};
 
 parameter_types! {
 	pub const KsmRelayLocation: MultiLocation = MultiLocation::parent();
-	pub const RelayNetwork: Option<NetworkId> = Some(NetworkId::Kusama);
+	pub const RelayNetwork: NetworkId = NetworkId::Kusama;
```

Comment:
How did this even manage to compile? `AccountId32Aliases` for example requires this to be an `Option<NetworkId>`.

Reply:
hmm, interesting, checking, CI passed with almost latest polkadot's `gav-xcm-v3`

https://gitlab.parity.io/parity/mirrors/cumulus/-/jobs/2190503
```
Compiling xcm-builder v0.9.33 (https://github.com/paritytech//polkadot?branch=gav-xcm-v3#4ccf61a6)
```

strange, for example contracts-rococo has `pub const RelayNetwork: Option<NetworkId> = None;` and others have `pub const RelayNetwork: NetworkId` and both compiles

maybe some blanket/macro hidden magic?